cardpopup.js

```diff
@@ -0,0 +1,282 @@
+    technology: ['html', 'css', 'javaScript'],
+  },
+
+  {
+    id: 3,
+    header: 'Multi-Post Stories',
+    tag1: 'CANOPY',
+    tag2: 'Back End Dev',
+    tag3: '2015',
+    extendedImage: './images/Snapshoot Portfolio.png',
+    content: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s,<br><br> when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technology: ['html', 'css', 'javaScript'],
+  },
+];
+
+const initInfo = [
+  {
+    id: 0,
+    header: 'Tonic',
+    tag1: 'CANOPY',
+    tag2: 'Back End Dev',
+    tag3: '2015',
+    extendedImage: './images/desktop/Snapshoot-Desktop1.png',
+    content: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technology: ['html', 'css', 'javaScript'],
+  },
+
+  {
+    id: 1,
+    header: 'Multi-Post Stories',
+    tag1: 'CANOPY',
+    tag2: 'Back End Dev',
+    tag3: '2015',
+    extendedImage: './images/snapshots/Snapshoot Portfolio 2.png',
+    content: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technology: ['html', 'css', 'javaScript'],
+  },
+
+  {
+    id: 2,
+    header: 'Tonic',
+    tag1: 'CANOPY',
+    tag2: 'Back End Dev',
+    tag3: '2015',
+    extendedImage: './images/snapshots/Snapshoot Portfolio 3.png',
+    content: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technology: ['html', 'css', 'javaScript'],
+  },
+
+  {
+    id: 3,
+    header: 'Multi-Post Stories',
+    tag1: 'CANOPY',
+    tag2: 'Back End Dev',
+    tag3: '2015',
+    extendedImage: './images/snapshots/Snapshoot Portfolio 4.png',
+    content: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technology: ['html', 'css', 'javaScript'],
+  },
+];
+
+cards.innerHTML = ` 
+<div class="secondBody">
+  <div class="secondImage swap">
+  <img src="${initInfo[0].extendedImage}" width="100%"/>       
+  </div>
+  <div>
+    <h2 class="second_title">${initInfo[0].header}</h2>
+    <ul class="second_subtitle">
+      <li class="remove">${initInfo[0].tag1}</li>
+      <li>${initInfo[0].tag2}</li>
+      <li>${initInfo[0].tag3}</li>
+    </ul>
+    <div class="second_info">
+      <p>
+      ${initInfo[0].content}
+      </p>
+      <ul class="parts">
+        <li>${initInfo[0].technology[0]}</li>
+        <li>${initInfo[0].technology[1]}</li>
+        <li>${initInfo[0].technology[2]}</li>
+      </ul>
```

Comment:
Good job working on the details popup window 👍, however, you still need to fix some part because you popup is not working properly according to the project requirement.

- [ ] If you check the project [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=When%20the%20user%20clicks%20(or%20taps)%20the%20button%20to%20check%20project%20details%2C%20the%20popup%20with%20details%20about%20the%20project%20appears.), When the user clicks (or taps) the button (`see project` button) to check project details, the popup with details about the project appears, then When the user clicks (or taps) the close `(X)` button, the popup disappears. it suppose to display this : 

> | | [For desck](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-1143&scaling=min-zoom&page-id=0%3A1) (click there) | | [For mobile](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-418&scaling=min-zoom&page-id=0%3A1) (click there) | |
> |-|---------|-|--------|-|
> | |![Capture](https://user-images.githubusercontent.com/57408419/232258287-faa1836e-07b7-4426-ad95-9dd48aa7933b.PNG)| |![Capture](https://user-images.githubusercontent.com/57408419/232258341-e952049a-f77f-4388-8a6d-116aa18028a6.PNG)| | 

# Then

- [ ] Make sure you keep the some design as the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-848&t=qS10UYrYRV4wxfyp-0). then when click on `see project` button, it'll give you the the popup like those image above 🆙 

- [ ] Kindly make sure to update the main page so that the projects section is created dynamically (`name`, `description`, `featured image`, `technologies`, `link to live version`, `link to source`.) using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20update%20the%20main%20page%20so%20that%20the%20projects%20section%20is%20created%20dynamically%20using%20the%20information%20stored%20in%20that%20JavaScript%20object.%20Remember%2C%20all%20of%20the%20HTML%20in%20that%20section%20is%20created%20when%20the%20page%20loads.).